Small improvements to the documentation (#508)
brefphp/bref: docs/websites.md

Feel free to customize the `/asset/` path. If your application is a JS application backed by a PHP API, you will want to invert API Gateway and S3 (set S3 as the `DefaultCacheBehavior` and serve API Gateway under a `/api/` path).

> > The first deployment takes a lot of time (20 minutes) because CloudFront is a distributed service. The next deployments that do not modify CloudFront's configuration will not suffer from this delay. You will know it is ready because the `Status` column will change from `In Progress` to `Deployed` on your [CloudFront dashboard](https://console.aws.amazon.com/cloudfront/home?region=eu-west-1#)

Reviewer: ```suggestion
> The first deployment takes a lot of time (20 minutes) because CloudFront is a distributed service. The next deployments that do not modify CloudFront's configuration will not suffer from this delay. You will know it is finished when the `Status` column changes from `In Progress` to `Deployed` in your [CloudFront dashboard](https://console.aws.amazon.com/cloudfront/home).
```

Author: @mnapoli The URL with the `?region=eu-west-1` I got it from here, a bit further down. Should I hange that too?

Reviewer: Yes that would be perfect 👍

Author: Done :-)

The URL of the deployed CloudFront distribution can be found in [the CloudFront dashboard](https://console.aws.amazon.com/cloudfront/home?region=eu-west-1#) under `Domain Name`.

### Setting up a domain name

> > Please note: Adding these settings after your first deployment will be a change of CloudFront's configuration and you will need to wait for ~20 minutes for it to be deployed. 

Reviewer: I don't think we should reiterate that as it is covered by the note above.

Just like in the "[Custom domains](/docs/environment/custom-domains.md)" guide, you need to register your domain in **ACM** (AWS Certificate Manager) to get a HTTPS certificate.

> > Please note: If you have already configured this domain in the API Gateway > Custom Domains (on any region), you will need to remove it from there before continuing. Otherwise, your application will still be served from there (and will not be using CloudFront)

Reviewer: I tried to make it simpler:

```suggestion
> If you have already set up this domain as a custom domain in API Gateway (by following the [Custom domain](/docs/environment/custom-domains.md) guide), you will need to remove it before continuing.
```
